vignettes/Using_rerddap.Rmd

```diff
@@ -0,0 +1,671 @@
+---
+title: "Using rerddap to Access Data from ERDDAP Servers"
+author: "Roy Mendelssohn and Scott Chamberlain"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rerddapVignette}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+```{r initialize, echo = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library(rerddap)
+```
+---
+
+## Introduction
+
+`rerddap` is a general purpose <span style="color:blue">R</span> client for working with <span style="color:blue">ERDDAP</span> servers.  <span style="color:blue">ERDDAP</span> is a web service developed by Bob Simons of NOAA.  At the time of this writing, there are over fifty <span style="color:blue">ERDDAP</span> servers  (though not all are public facing) providing access to literally petabytes of data and model output relevant to oceanography, meteorology, fisheries and marine mammals, among other areas. <span style="color:blue">ERDDAP</span> is a simple to use, RESTful web service, that allows data to be subsetted and returned in a variety of formats.
+
+In this vignette we go over some of the nuts and bolts of using the `rerddap` package, and show the power of the combination of the `rerddap` package with <span style="color:blue">ERDDAP</span> servers.  Some of the examples are taken from the `xtractomatic` package (available from CRAN), and some from the `rerddapXtracto` package available on Github,  but reworked to use `rerddap` directly.   Other examples are new to this vignette, and include both gridded and non-gridded datasets from several <span style="color:blue">ERDDAPs</span>.
+
+## Installation
+
+The first step is to install the `rerddap` package, the stable version is available from CRAN:
+
+```{r, eval = FALSE}
+install.packages("rerddap")
+```
+
+or the development version can be installed from GitHub:
+
+```{r, eval = FALSE}
+devtools::install_github("ropensci/rerddap")
+```
+
+and to load the library:
+
+```{r}
+library("rerddap")
+load("colors.rda")
+```
+
+Besides `rerddap` the following libraries are used in this vignette:
+
+```{r, warning = FALSE}
+library("akima")
+library("dplyr")
+library("ggfortify")
+library("ggplot2")
+library("lubridate")
+library("mapdata")
+library("ncdf4")
+library("parsedate")
+library("plot3D")
+library("xts")
+```
```

Comment:
There's a bunch of messages/et.c when these pkgs load, e.g., 

```r
library("akima")
library("dplyr")
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union
library("ggfortify")
```

maybe duplicate this block of code, and with one of them just set to `eval=FALSE` so it doesn't run, but is shown to reader, and with the other one, add `warn.conflicts = FALSE` and  `quietly = TRUE` (or maybe just `quietly = TRUE`)